### DISCUSSION TITLE
Anti-Kick Invisible Message Code

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#316] [Q&A] [answered]
Hey! its me again xd.
Just wondering is it possible to fully hide the join message? i tried to play around with the codes but all those changes i edited just made it sending message "[Playername] joined the ship."

I dont actually know if that message are server-sided or client-sided

**Comment by winstxnhdw** [comment, marked as answer]
You can just change `Chat.Announce($"{Helper.LocalPlayer?.playerUsername} disconnected.", true);` to `Chat.Announce("");`

**Reply by Dexteriority** [reply]
Thanks!
